pkg/services/dashboards/folder_service_test.go

```diff
@@ -194,9 +195,8 @@ func TestFolderService(t *testing.T) {
 
 			for _, tc := range testCases {
 				actualError := toFolderError(tc.ActualError)
-				if actualError != tc.ExpectedError {
-					t.Errorf("For error '%s' expected error '%s', actual '%s'", tc.ActualError, tc.ExpectedError, actualError)
-				}
+				require.EqualErrorf(t, actualError, tc.ExpectedError.Error(),
```

Comment:
```suggestion
				assert.EqualErrorf(t, actualError, tc.ExpectedError.Error(),
```

To make sure all failures are listed.